pkg/parser/model/ddl.go

```diff
@@ -585,15 +585,36 @@ type Job struct {
 	SQLMode mysql.SQLMode `json:"sql_mode"`
 }
 
-// InvolvingSchemaInfo returns the schema info involved in the job.
-// The value should be stored in lower case.
+// InvolvingSchemaInfo returns the schema info involved in the job. The value
+// should be stored in lower case. The top level fields are exclusively used for
+// the job, and the shared objects are saved in Shared field.
+//
+// Exclusive and shared objects are considered like the exclusive lock and shared
+// lock when calculate DDL job dependencies. And we also implement the fair lock
+// semantic which means if job B (exclusive request object 0) is waiting for the
+// running job A (shared request object 0), and job C (shared request object 0)
+// arrives, job C should also be blocked until job B is finished although job A &
+// C has no dependency.
 type InvolvingSchemaInfo struct {
-	Database string `json:"database"`
-	Table    string `json:"table"`
+	Database      string                     `json:"database"`
+	Table         string                     `json:"table"`
```

Comment:
```suggestion
	Database      string                     `json:"database,omitempty"`
	Table         string                     `json:"table,omitempty"`
```

Reply:
Not sure we should keep compatibility, but seems no harm.